### ISSUE TITLE
Features for v0.5.0

### ISSUE THREAD

**Issue by ehzhang** [facebook/fixed-data-table#193]
[ ] Reformat API to use React components instead of rowGetters and cellRenderers

**Comment by cosminnicula** [comment]
Any chance to add accessibility for this release? :)

**Comment by ehzhang** [comment]
Hey @cosminnicula!

Right now, the focus is on a new API design that will make it much easier to create your own components for the table and maximize flexibility. Once we have stabilized on that API, it should be easier to focus on accessibility!

**Comment by rmilesson** [comment]
What's the status for this release? We're using FDT extensively and this would be a very neat addition.

**Comment by ehzhang** [comment]
@rmilesson this release is currently still in development - we're also working to make sure that the migration goes smoothly before making such large API changes.

**Comment by box-turtle** [comment]
Will this new API make it possible to use a menu or dropdown component as a cell such  that the dropdown can escape the bounds of the cell?

**Comment by wlis** [comment]
@box-turtle I am not sure that this will come with the new API, but removing unnecessary overflow:hidden from many places in our CSS will also happen soon.

**Comment by box-turtle** [comment]
Awesome! Currently I'm able to work around using the portal technique suggested in another issue. Maybe removing overflow:hidden could allow us to simulate colspan as well.

**Comment by fobbyal** [comment]
@box-turtle can you paste the link of the of the protal technique?

**Comment by box-turtle** [comment]
I'm using a react-bootstrap menu inside a react-overlay component.

https://github.com/react-bootstrap/react-overlays


```
<Overlay
           show={this.state.show}
           onHide={() => this.setState({ show: false })}
           placement="bottom"
           rootClose
           container={document.body}
           target={ () => ReactDOM.findDOMNode(this.refs.target)}
         >
            <ul className="dropdown-menu" style={{display: 'block'}}>
              <MenuItem key='freezeMenu' onSelect={this.freeze}>{col.fixed ? 'Unfreeze' : 'Freeze'}</MenuItem>
              {groupMenu}
            </ul>
        </Overlay>
```

**Comment by fobbyal** [comment]
@box-turtle thx
